test/decl/protocol/special/coding/struct_codable_simple_extension.swift

```diff
@@ -2,7 +2,7 @@
 
 // Simple structs where Codable conformance is added in extensions should still
```

Comment:
This comment does *not* match up with the implementation or expectation of the failure of this test case.